dev/com.ibm.ws.wssecurity_fat.wsscxf.1/com.ibm.ws.wssecurity.example.cbh.bnd

```diff
@@ -14,7 +14,7 @@
 bVersion=1.0.0
 
 Bundle-Name: wssecuritycbh
-Bundle-SymbolicName: com.ibm.ws.wssecurity.example.cbh
+Bundle-SymbolicName: com.ibm.ws.wssecurity.example.cbh.1
```

Comment:
You can't change the bundle symbolic name without changing the features that reference the bundle.  See wssecch-1.0.mf and wssecch-2.0.mf in each of the projects.  You also change change the name of the bnd file to match the symbolic name.

Reply:
You will also need to update tests where the files are copied and deleted from the liberty install root:

```
        if (features.contains("usr:wsseccbh-1.0")) {
            server.copyFileToLibertyInstallRoot("usr/extension/lib/", "bundles/com.ibm.ws.wssecurity.example.cbh.jar");
            server.copyFileToLibertyInstallRoot("usr/extension/lib/features/", "features/wsseccbh-1.0.mf");
            copyServerXml(System.getProperty("user.dir") + File.separator + server.getPathToAutoFVTNamedServer() + "server.xml");
        } else if (features.contains("usr:wsseccbh-2.0")) {
            server.copyFileToLibertyInstallRoot("usr/extension/lib/", "bundles/com.ibm.ws.wssecurity.example.cbhwss4j.jar");
            server.copyFileToLibertyInstallRoot("usr/extension/lib/features/", "features/wsseccbh-2.0.mf");
            copyServerXml(System.getProperty("user.dir") + File.separator + server.getPathToAutoFVTNamedServer() + "server_wss4j.xml");
        }
```
for example.  Just search for `com.ibm.ws.wssecurity.example.cbh` in the java files to find where things need to be changed.

Reply:
Thanks for catching. I'll include these changes soon.

Reply:
I've made the suggested changes in a commit, but I've found an issue with this approach. The common test project `com.ibm.ws.security.saml.sso_fat.common` refers to the Bundle-SymbolicName in a constant and is presumably used by each of the others, so would need to have the number extension too.
https://github.com/OpenLiberty/open-liberty/blob/9ef2c4571a910e7e64f8159181aad076f7e377ee/dev/com.ibm.ws.security.saml.sso_fat.common/src/com/ibm/ws/security/saml20/fat/commonTest/SAMLConstants.java#L556

Reason I've added a `.#` extension to the Bundle-SymbolicName is because when running in parallel, the release task fails to publish stating that there is already a released bundle. I haven't hit this problem on any of the other bundles you mentioned, but it seems safe for them also to be uniquely named.

Reply:
And projects `com.ibm.ws.wssecurity_fat.wsscxf.saml.1` through `com.ibm.ws.wssecurity_fat.wsscxf.saml.6` do not hit this issue because they have the test bundle checked directly into source control, which should get fixed up to pull from an artifact in a repo instead. I will try to separate the `com.ibm.ws.wssecurity.example.cbh` and `com.ibm.ws.wssecurity.example.cbhwss4j` test bundles into their own projects so there's only 1 artifact of each being released.